packages/client/components/DiscussionThread.tsx

```diff
@@ -28,16 +30,18 @@ const Wrapper = styled('div')<{isExpanded: boolean}>(({isExpanded}) => ({
 
 interface Props {
   meetingContentRef: RefObject<HTMLDivElement>
+  threadSourceId: string
   viewer: DiscussionThread_viewer
 }
 
 const DiscussionThread = (props: Props) => {
-  const {meetingContentRef, viewer} = props
+  const {meetingContentRef, threadSourceId, viewer} = props
   const meeting = viewer.meeting!
   const {endedAt, replyingToCommentId, threadSource} = meeting
-  const {commentors, thread} = threadSource!
-  const threadSourceId = threadSource!.id!
-  const preferredNames = commentors && commentors.map((commentor) => commentor.preferredName)
+  const thread = threadSource?.thread
```

Comment:
I came across a strange bug where `threadSource` was null and an error was thrown. I ended a check-in meeting, clicked into it from the Timeline and went to the Discuss stage where the bug was thrown. I've created several different accounts and tried many combinations of ending a meeting but I haven't been able to repro it.

Line 45 mentions an unusual Relay error relating to `thread` that Terry reported. Perhaps this is the same issue or perhaps something strange happened with the gql executor? If you login with user: t1@lisboa.com pass: testing you can find the bug in Check-in #5 on the first agenda item but not the subsequent 3. The `threadSourceId` is invalid. 

To prevent the app from crashing I made this change of line 41 and pass the `threadSourceId` down from the parent but it's not possible to add a comment/task to this agenda item.